apps/meteor/client/hooks/omnichannel/useIsRoomOverMacLimit.tsx

```diff
@@ -4,6 +4,7 @@ import { isOmnichannelRoom, type IOmnichannelGenericRoom, isVoipRoom } from '@ro
 import { useIsOverMacLimit } from './useIsOverMacLimit';
 
 const getPeriod = (date: Date) => `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(2, '0')}`;
+const isILivechatInquiry = (room: any) => room.rid && room.status;
```

Comment:
The worst type guard ever! Why `any`? Why `rid` and `status`? Why not asserting the type? <img src="https://media3.giphy.com/media/YpmVBNubONoqs/giphy.gif"/>